### PR TITLE
FLW-2534: New alert for app

### PR DESCRIPTION
**Pull request by DRadmir**

### CODE REVIEW

File: fearless/Common/Protocols/SheetAlertPresentable.swift [hunk 1]
```diff
@@ -56,4 +76,34 @@ extension SheetAlertPresentable {
...
+    func present(
+        message: String?,
+        title: String,
+        closeAction: String?,
+        from view: ControllerBackedProtocol?
+    ) {
+        var currentController = view?.controller
+
+        if currentController == nil {
+            currentController = UIApplication.shared.delegate?.window??.rootViewController
```
Comment (bnsports) [inline fearless/Common/Protocols/SheetAlertPresentable.swift:89]: no good. can we create a new uiwindow to show alert? that's default apple practice

File: fearless/Common/ViewController/NavigationController/ImportantFlowNavigationController.swift [hunk 2]
```diff
@@ -37,20 +37,20 @@ extension ImportantFlowNavigationController: UIAdaptivePresentationControllerDel
     func presentationControllerDidAttemptToDismiss(_: UIPresentationController) {
         let languages = localizationManager.selectedLocale.rLanguages
 
-        let action = AlertPresentableAction(
+        let action = SheetAlertPresentableAction(
             title: R.string.localizable.commonCancelOperationAction(preferredLanguages: languages),
-            style: .destructive
+            style: UIFactory.default.createDestructiveButton()
```
Comment (bnsports) [inline fearless/Common/ViewController/NavigationController/ImportantFlowNavigationController.swift:42]: what's this? style or button? I don't understand

File: fearless/Common/ViewController/SheetAlertViewLayout.swift [hunk 3]
```diff
@@ -58,11 +60,24 @@ final class SheetAlertViewLayout: UIView {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        closeButton.rounded()
```
Comment (bnsports) [inline fearless/Common/ViewController/SheetAlertViewLayout.swift:65]: applyRoundedStyle() or round() or roundOff()

File: fearless/Modules/AccountImport/AccountImportPresenter.swift [hunk 4]
```diff
@@ -470,28 +471,26 @@ private extension AccountImportPresenter {
         }
     }
 
-    func askIfNeedAddEthereum(showHandler: @escaping () -> Void, closeHandler: @escaping () -> Void) {
-        let showAction = AlertPresentableAction(
+    func askIfNeedAddEthereum(showHandler: @escaping () -> Void, closeHandler _: @escaping () -> Void) {
```
Comment (bnsports) [inline fearless/Modules/AccountImport/AccountImportPresenter.swift:474]: unused parameter 'closeHandler'

**Review by bnsports — CHANGES_REQUESTED** [review]

File: fearless/Common/ViewController/SheetAlertViewLayout.swift (hunk 3 above)
Comment (DRadmir) [reply fearless/Common/ViewController/SheetAlertViewLayout.swift:65]: closeButton is UIButton

**Review by bnsports — APPROVED** [review]

**Pull request merged**